bip-tap-universe.mediawiki

```diff
@@ -253,8 +253,11 @@ A Multiverse has the following structure:
 ** <code>value</code>: <code>universe_proof_leaf</code>
 ** <code>sum_value</code>: the total amount of asset units issued by the proof leaf.
 
-A Multiverse therefore commits to the set of known genesis IDs, and at a second
-level the set of complete Universe trees for each watched asset.
+A Multiverse therefore commits to the set of known genesis IDs, and at a second level the set of complete Universe trees
```

Comment:
Hmm, isn't this clear with the above? In that the leaves of the upper tree are the roots of the lower tree?

Reply:
In the latest commits, I've removed the additional comments.